core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -223,20 +223,27 @@ public TableScan ignoreResiduals() {
     } else {
       splitSize = targetSplitSize();
     }
+    Preconditions.checkArgument(splitSize > 0, "Split size should be greater than zero. Found: %s", splitSize);
```

Comment:
We normally phrase exception messages like "Invalid split size (negative): %s". That says basically the same thing, but is shorter and follows the conventions we use elsewhere.

Reply:
@shardulm94 could you update the error messages here?